cleanlab/datalab/data_issues.py

```diff
@@ -22,20 +22,21 @@
 instances and keeps track of each issue, a summary for each type of issue,
 related information and statistics about the issues.
 
-The collected information can be accessed using the 
+The collected information can be accessed using the
 :py:meth:`get_info <cleanlab.datalab.data_issues.DataIssues.get_info>` method.
```

Comment:
```suggestion
The collected information can be accessed using the
:py:meth:`get_info <cleanlab.datalab.data_issues.DataIssues.get_info>` method.
We recommend using that method instead of this module, which is just intended for internal use.
```